.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
This file is editor-specific. Please remove it and add `.idea/` to the `.gitignore` file

Reply:
Please create an `editorconfig` file as described on https://editorconfig.org/ instead if you want to add code guidelines. This works cross-editor.

Reply:
done